cns/configuration/configuration.go

```diff
@@ -220,5 +223,12 @@ func SetCNSConfigDefaults(config *CNSConfig) {
 	if config.AsyncPodDeletePath == "" {
 		config.AsyncPodDeletePath = "/var/run/azure-vnet/deleteIDs"
 	}
+	if config.GRPCServerIPAddress == "" {
+		config.GRPCServerIPAddress = "::"
+	}
+	if config.GRPCServerPort == 0 {
+		config.GRPCServerPort = 8080
+	}
+	config.EnableGRPC = true
```

Comment:
This should not be checked in. The default should be `False`

Reply:
Right sorry about that